mat64/cholesky.go

```diff
@@ -79,8 +79,7 @@ func (t *TriDense) Cholesky(a *SymDense, upper bool) (ok bool) {
 }
 
 // SolveCholesky finds the matrix x that solves A * X = B where A = L * L^T or
-// A = U^T * U, and U or L are represented by t. The matrix A must be symmetric
-// and positive definite.
+// A = U^T * U, and U or L are represented by t.
 func (m *Dense) SolveCholesky(t Triangular, b Matrix) {
```

Comment:
Dense receivers are almost consistently called 'm' but for example in this case wouldn't it be better to relate better the function signature with the comment? The comment talks about x, X, A, B, L, U, while the function takes m, t, b.